.github/templates/main.yml

```diff
@@ -72,6 +73,34 @@ steps:
       retention-days: 1
 #@ end
 
+#@ def buildNugetPkgs():
+#@ for pkgName in nugetPackages:
+#@ buildCmd = "MsBuild.exe ${{ github.workspace }}/Realm/" + pkgName + " -t:Pack -p:Configuration=" + configuration + " -p:PackageOutputPath=${{ github.workspace }}/Realm/packages -p:VersionSuffix=${{ steps.set-version-suffix.outputs.BUILD_SUFFIX }} -restore"
+  - name: #@ "Build nuget package " + pkgName
+    run: #@ buildCmd
+#@ end
+
+  - name: Find nupkg version
+    id: find-nupkg-version
+    run: |
+      cd Realm/packages
+      tmpName=$(basename Realm.Fody* .nupkg)
+      pkgName=${tmpName#"Realm.Fody."}
+      echo "::set-output name=NUGET_PKG_VERSION::$pkgName"
+    shell: bash
+
+#@ for pkgName in nugetPackages:
+#@ finalPkgName = "${{ steps.set-version-suffix.outputs.BUILD_SUFFIX }}" + pkgName + ".${{ steps.find-nupkg-version.outputs.NUGET_PKG_VERSION }}.nupkg"
```

Comment:
The nupkg version should include the suffix, so there's no need to explicitly have it in the finalPkgName.